gammapy/data/data_store.py

```diff
@@ -711,3 +713,59 @@ def make_psf(self, position, energy=None, theta=None):
         psf_tot = EnergyDependentTablePSF(energy=energy, offset=theta, exposure=exposure,
                                           psf_value=psf_value.T)
         return psf_tot
+
+    def make_mean_rmf(self, position, e_true, e_reco, low_reco_threshold=None, high_reco_threshold=None):
+        r"""Make mean rmf for a given position and a set of observations.
+
+        Compute the mean rmf of a set of observations j at a given position
+
+        The stacking of :math:`j` observations is implemented as follows.  :math:`k`
+        and :math:`l` denote a bin in reconstructed and true energy, respectively.
+
+        .. math::
+
+            \epsilon_{jk} =\left\{\begin{array}{cl} 1, & \mbox{if
+                bin k is inside the energy thresholds}\\ 0, & \mbox{otherwise} \end{array}\right.
+
+            \overline{\mathrm{edisp}}_{kl} = \frac{\sum_{j} \mathrm{edisp}_{jkl}
+                \cdot \mathrm{aeff}_{jl} \cdot t_j \cdot \epsilon_{jk}}{\sum_{j} \mathrm{aeff}_{jl}
+                \cdot t_j}
+
+        Parameters
+        ----------
+        position : `~astropy.coordinates.SkyCoord`
+            Position at which to compute the PSF
```

Comment:
Copy & paste error: "PSF"-> "EDISP" (or put something shorter, less specific in both docstrings).